composer/workflows/airflow_db_cleanup.py

```diff
@@ -410,13 +396,11 @@ def cleanup_function(**context):
     try:
         if context["params"].get("do_not_delete_by_dag_id"):
             query = build_query(
-                session,
-                airflow_db_model,
-                age_check_column,
-                max_date,
-                keep_last,
-                keep_last_filters,
-                keep_last_group_by,
+                session=session,
+                airflow_db_model=airflow_db_model,
+                age_check_column=age_check_column,
+                max_date=max_date,
+                dag_id=None,
```

Comment:
IMHO passing here `dag_id` parameter seems a bit unreadable.
We anyway have it None as default in signature of the method, and since this parameter shouldn't be used for other tables than DagRun then it can be misleading to pass it?

Reply:
Agree.